dispatch/BESError.h

```diff
@@ -104,6 +112,16 @@ class BESError: public std::exception,  public BESObj {
         _msg = msg;
     }
 
+    /**
+     * Used to add error specific details to the BESInfo object
+     * @param info
+     */
+    virtual void add_my_error_details_to(BESInfo &info) const {
+        // Some errors are covered by the basic details and the message.
+        // Others, like HttpError require more information to be carried
+        // in the error so the override this method.
```

Comment:
> so the override this method.

Reply:
Not quite understand the meaning of the above phrase.

Reply:
I rewrote it.